app/models/badges/chatterbox_badge.rb

```diff
@@ -0,0 +1,12 @@
+module Badges
+  class ChatterboxBadge < Badge
+    seed "Chatterbox",
+      :common,
+      :chatterbox,
+      "Linked your Exercism account to Discord"
+
+    def award_to?(user) = user.discord_uid.present?
+
+    def send_email_on_acquisition? = false
```

Comment:
Not sure if we wanted to send an email

Reply:
I'd say `false` in general, but I'd like to email it when we batch award it. So maybe deploy it as true, do the batch, then change it to false. Same for the forum one.